app/Controllers/entryController.php

```diff
@@ -169,7 +169,9 @@ public function optimizeAction(): void {
 			Minz_Request::forward($url_redirect, true);
 		}
 
-		@set_time_limit(300);
+                if (function_exists('set_time_limit')) {
+                        @set_time_limit(300);
```

Comment:
```suggestion
                       $isTimeLimited = set_time_limit(300);
                       if ($isTimeLimited === false) {
                                    continue;
                       }
```

Reply:
You can add the logic in function

Reply:
@ColonelMoutarde After PHP8, with function set_time_limit disabled, set_time_limit (or even decorated with @) will cause fatal termination of script.

Reply:
Hi, https://www.php.net/manual/en/function.set-time-limit.php set_time_limit is available for all php versions from 4 to 8.
The @ is not necessary, especially as you're checking that the function is present just beforehand. ;)

Reply:
> set_time_limit is available for all php versions from 4 to 8.

Partially wrong: any function can be disabled, and we have several examples of providers doing so for that specific function

> The @ is not necessary

Wrong: there might be a warning with PHP < 8. See https://php.watch/versions/8.0/disable_functions-redeclare